sources/api/controllers/UserController.ts

```diff
@@ -222,3 +222,35 @@ export async function getUserGroups(req: Request, res: Response) {
         groups: UserView.formatUserGroupList(groups)
     });
 }
+
+export async function getUserCalendars(req: Request, res: Response) {
+    return res.status(200).json({
+        message: "OK",
+        calendars: [
+            {
+                calendar_id: 2,
+                owner_id: 1,
```

Comment:
What is `owner_id` ?

Reply:
my bad that was not supposed to be visible